src/main/python/pybuilder/plugins/python/coverage_plugin.py

```diff
@@ -39,6 +39,8 @@ def init_coverage_properties(project):
     project.build_depends_on("coverage")
 
     project.set_property_if_unset("coverage_threshold_warn", 70)
+    project.set_property_if_unset("coverage_branch_threshold_warn", 0)
```

Comment:
I was thinking we could emit a warning when the threshold is at 0, so users who do not follow release notes would note the new features and adjust the threshold (personally I think branch coverage capability is HUGE).
But OTOH it's okay to set it to 0% if you want to just "know" what your coverage is when testing, and don't want to act on it... I'm torn between both viewpoints which are both important, I think... what's your take on this?